src/ParallelAlgorithms/Events/ObserveTimeStepVolume.hpp

```diff
@@ -60,60 +70,79 @@ namespace dg::Events {
  * - Time step
  * - Slab fraction
  * - Minimum grid spacing
+ * - Integration order
  */
-template <size_t VolumeDim>
-class ObserveTimeStepVolume : public ObserveConstantsPerElement<VolumeDim> {
+template <typename System>
+class ObserveTimeStepVolume
```

Comment:
Could we put this in a tpp file and do explicit instantiations for systems? We already compile soooo much code in the final executable. Another option would be to have a compute tag we add into the observation box that gets the order from the history and then this can just retrieve the simple tag.

Reply:
Sounds great, thank you!